tests/test_databases.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import trio
+
+from typing import AsyncContextManager
+
+from piker._daemon import Services
+from piker.log import get_logger
+
+from elasticsearch import Elasticsearch
+from piker.data import marketstore
+
+def test_marketstore(
```

Comment:
maybe change the names of these tests to be something like `test_<dbname>_startup_and_version()` ? Just so we can distinguish them from other more (presumably) specific tests in the future?